docs/writing-stories/build-pages-with-storybook.md

```diff
@@ -394,3 +394,22 @@ If you’ve set up `GlobalContainerContext`, you’ll need to set up a decorator
 />
 
 <!-- prettier-ignore-end -->
+
+### Mocking fonts
```

Comment:
Not sure if this is the right place to put this, but it seems like this file is all about mocks

Reply:
@kylegach are you saying @nsheaps should move all of the changes into that README.md file? Or maybe in the [FAQ](https://storybook.js.org/docs/react/faq)?

Reply:
Yes, into the README. For now. We'll base the better-located docs on those READMEs when the time comes.

Reply:
I'll take care of that in the upcoming days, thanks for the feedback!